examples/multiple-daemons/dataflow.yml

```diff
@@ -4,7 +4,7 @@ nodes:
       machine: A
     custom:
       build: cargo build -p multiple-daemons-example-node
-      source: ../../target/debug/multiple-daemons-example-node
+      source: /home/runner/work/dora/dora/target/debug/multiple-daemons-example-node
```

Comment:
Could be good to point out that this is for Github Action and that people should adapt this to the remote machine path :)